utils/zoneinfo64/src/lib.rs

```diff
@@ -287,7 +299,7 @@ pub enum PossibleOffset {
     // <https://tc39.es/proposal-temporal/#sec-getnamedtimezoneepochnanoseconds>
     Ambiguous(Offset, Offset),
     /// There is no possible offset, this is a gap transition
-    None,
+    None(GapOffsets),
```

Comment:
not a fan of the `GapOffsets` name, maybe we don't need the type?

```suggestion
    None { 
      before: UtcOffset,
      after: UtcOffset,
      transition: i64,
    },
```